packages/flutter/test/painting/shader_warm_up_test.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' as ui;
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+Future<void> main() async {
```

Comment:
this test should fail when running on web in HTML mode. you need to add a `skip: kIsWeb && !kIsCanvasKit`